src/vue/backend/src/store/modules/general/index.js

```diff
@@ -3,6 +3,9 @@ import getters from './getters'
 import mutations from './mutations'
 
 const state = {
+    reCAPTCHA: false,
+    updateFrequency: { value: '', label: '' },
+    wrapperName: 'idx-wp-'
```

Comment:
In the existing plugin, the default wrapper name is unset, so this should be an empty string